### ISSUE TITLE
warning about yaml not updated

### ISSUE THREAD

**Issue by avikivity** [scylladb/scylla-cluster-tests#7154]
### Issue description

- [ ] This issue is a regression.
- [x] It is unknown if this issue is a regression.

#### Steps to Reproduce

run s-c-t 650GB mixed with nemesis

get this warning

```
21:33:54  < t:2024-01-27 19:33:52,218 f:cluster.py      l:4333 c:sdcm.cluster         p:INFO  > Cluster perf-latency-grow-shrink-ubuntu-db-cluster-c8bc6de8 (AMI: ['ami-00180dd44d74fc6ec'] Type: i3en.2xlarge): Waiting for preinstalled Scylla
21:33:54  < t:2024-01-27 19:33:52,219 f:cluster.py      l:2927 c:sdcm.cluster_aws     p:INFO  > Node perf-latency-grow-shrink-ubuntu-db-node-c8bc6de8-2 [54.154.228.207 | 10.4.3.101] (seed: False): Waiting for Scylla Machine Image setup to finish...
21:33:54  < t:2024-01-27 19:33:52,264 f:cluster.py      l:4335 c:sdcm.cluster         p:INFO  > Cluster perf-latency-grow-shrink-ubuntu-db-cluster-c8bc6de8 (AMI: ['ami-00180dd44d74fc6ec'] Type: i3en.2xlarge): Done waiting for preinstalled Scylla
21:33:54  < t:2024-01-27 19:33:52,264 f:logcollector.py l:821  c:sdcm.logcollector    p:INFO  > Saving kallsyms map from host: perf-latency-grow-shrink-ubuntu-db-node-c8bc6de8-2
21:33:55  < t:2024-01-27 19:33:55,373 f:base.py         l:320  c:sdcm.sct_events.base p:WARNING > [SCT internal warning] (ScyllaYamlUpdateEvent Severity.NORMAL) period_type=one-time event_id=adcdae09-7c6b-4b69-bc53-d5ec6ec39dd9: message=ScyllaYaml has been changed on node: perf-latency-grow-shrink-ubuntu-db-node-c8bc6de8-2. Diff: --- 
21:33:55  < t:2024-01-27 19:33:55,373 f:base.py         l:320  c:sdcm.sct_events.base p:WARNING > +++ 
```

Does it mean the configuration was not updated?

**Comment by fruch** [comment]
we need to change the level of this print, it mean it loaded the yaml and no changes were done to it.

it's a code that was built for changing scylla.yaml mostly, hence the warning.
but since then it's been used for reading scylla.yaml, and hence the warnings

**Comment by fruch** [comment]
was fixed in #5893, needs to be backport into v14
